frontend/src/ts/controllers/sound-controller.ts

```diff
@@ -269,43 +275,43 @@ function bindToNote(
 }
 
```

Comment:
Change the type signature for `bindToNote`

```suggestion
function bindToNote(
  noteFrequencies: ValidFrequencies,
  octaveOffset = 0
): GetNoteFrequencyCallback {
  return (octave: number): number => {
    return noteFrequencies[octave + octaveOffset];
  };
}
```